cmd/ocm/login/cmd.go

```diff
@@ -153,6 +163,16 @@ func run(cmd *cobra.Command, argv []string) error {
 		return fmt.Errorf("Option '--url' is mandatory")
 	}
 
+	if args.useAuthCode {
+		fmt.Println("You will now be redirected to Red Hat SSO login")
+		token, err := authentication.VerifyLogin(oauthClientID)
+		if err != nil {
+			return err
```

Comment:
Let's add some additional context here. At minimum indicating that we failed to login and appending the error.

Reply:
Added message before the error